storage/Filesystems/ExfatImpl.cc

```diff
@@ -0,0 +1,100 @@
+/*
+ * Copyright (c) 2018 SUSE LLC
+ *
+ * All Rights Reserved.
+ *
+ * This program is free software; you can redistribute it and/or modify it
+ * under the terms of version 2 of the GNU General Public License as published
+ * by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+ * FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License for
+ * more details.
+ *
+ * You should have received a copy of the GNU General Public License along
+ * with this program; if not, contact Novell, Inc.
+ *
+ * To contact Novell about this file by physical or electronic mail, you may
+ * find current contact information at www.novell.com.
+ */
+
+
+#include "storage/Utils/StorageDefines.h"
+#include "storage/Utils/SystemCmd.h"
+#include "storage/Devices/BlkDeviceImpl.h"
+#include "storage/Filesystems/ExfatImpl.h"
+#include "storage/FreeInfo.h"
+#include "storage/UsedFeatures.h"
+
+
+namespace storage
+{
+
+    using namespace std;
+
+
+    const char* DeviceTraits<Exfat>::classname = "Exfat";
+
+
+    Exfat::Impl::Impl(const xmlNode* node)
+	: BlkFilesystem::Impl(node)
+    {
+    }
+
+
+    string
+    Exfat::Impl::get_pretty_classname() const
+    {
+	// TRANSLATORS: name of object
```

Comment:
Name of "object"?? If I'm the translator, this is useless.
What is the context? What are other examples of "objects"?

Reply:
Apparently the correct spelling is "exFAT": https://en.wikipedia.org/wiki/ExFAT